nova/conf/mks.py

```diff
@@ -13,6 +13,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
+import os
```

Comment:
Is that import needed? I am not seeing a reference to it in the diff

Reply:
Forgot to run flake8, sorry about that. :/